.github/workflows/build-and-test.yaml

```diff
@@ -11,7 +11,7 @@ on:
     paths:
 
 env:
-  BENV_IMAGE: quay.io/splunko11ytest/network-explorer-debug/build-env
+  BENV_IMAGE: public.ecr.aws/u7d6c4a3/solarwinds-opentelemetry-network:build-env
```

Comment:
I believe you should move PR to draft if you are still experimenting with the changes. We shouldn't merge to upstream use of image from solarwinds owned registry